ibis/expr/types/relations.py

```diff
@@ -1656,7 +1656,8 @@ def relabel(
         substitutions: Mapping[str, str]
         | Callable[[str], str | None]
         | str
-        | Literal["snake_case"],
+        | Literal["snake_case"]
+        | Literal["ALL_CAPS"],
```

Comment:
```suggestion
        | Literal["snake_case", "ALL_CAPS"],
```

Reply:
Thank you! Sorry, I forgot to resolve this in the midst of the squashing. I just amended the commit with this change so hopefully we're set now!